jwst/refpix/reference_pixels.py

```diff
@@ -63,47 +63,63 @@
 # The 'stop' values are one more than the actual final row or column, in
 # accordance with how Python slices work
 
-NIR_reference_sections = {'A': {'top': (2044, 2048, 0, 512),
-                                'bottom': (0, 4, 0, 512),
-                                'side': (0, 2048, 0, 4),
-                                'data': (0, 2048, 0, 512)},
-                          'B': {'top': (2044, 2048, 512, 1024),
-                                'bottom': (0, 4, 512, 1024),
-                                'data': (0, 2048, 512, 1024)},
-                          'C': {'top': (2044, 2048, 1024, 1536),
-                                'bottom': (0, 4, 1024, 1536),
-                                'data': (0, 2048, 1024, 1536)},
-                          'D': {'top': (2044, 2048, 1536, 2048),
-                                'bottom': (0, 4, 1536, 2048),
-                                'side': (0, 2048, 2044, 2048),
-                                'data': (0, 2048, 1536, 2048)}
-                          }
+NIR_reference_sections = {
+    "A": {
+        "top": (2044, 2048, 0, 512),
+        "bottom": (0, 4, 0, 512),
+        "side": (0, 2048, 0, 4),
+        "data": (0, 2048, 0, 512),
+    },
+    "B": {
+        "top": (2044, 2048, 512, 1024),
+        "bottom": (0, 4, 512, 1024),
+        "data": (0, 2048, 512, 1024),
+    },
+    "C": {
+        "top": (2044, 2048, 1024, 1536),
+        "bottom": (0, 4, 1024, 1536),
+        "data": (0, 2048, 1024, 1536),
+    },
+    "D": {
+        "top": (2044, 2048, 1536, 2048),
+        "bottom": (0, 4, 1536, 2048),
+        "side": (0, 2048, 2044, 2048),
+        "data": (0, 2048, 1536, 2048),
+    },
+}
 
 # IRS2 sections for NIRSpec have a different size due to the
 # interleaved reference pixels and the reference sector.
 
-IRS2_reference_sections = {'0': {'top': (2044, 2048, 0, 640),
-                                 'bottom': (0, 4, 0, 640),
-                                 'data': (0, 2048, 0, 640)},
-                           'A': {'top': (2044, 2048, 640, 1280),
-                                 'bottom': (0, 4, 640, 1280),
-                                 'data': (0, 2048, 640, 1280)},
-                           'B': {'top': (2044, 2048, 1280, 1920),
-                                 'bottom': (0, 4, 1280, 1920),
-                                 'data': (0, 2048, 1280, 1920)},
-                           'C': {'top': (2044, 2048, 1920, 2560),
-                                 'bottom': (0, 4, 1920, 2560),
-                                 'data': (0, 2048, 1920, 2560)},
-                           'D': {'top': (2044, 2048, 2560, 3200),
-                                 'bottom': (0, 4, 2560, 3200),
-                                 'data': (0, 2048, 2560, 3200)}
-                           }
+IRS2_reference_sections = {
+    "0": {"top": (2044, 2048, 0, 640), "bottom": (0, 4, 0, 640), "data": (0, 2048, 0, 640)},
```

Comment:
I think in this case the autoformatter made this less legible, would you put these back on separate lines then add a format ignore for this block?

Reply:
I don't like the black formatting here either, so I # fmt: off those sections